content/en/serverless/google_cloud_run/_index.md

```diff
@@ -46,6 +46,10 @@ ENV DD_SERVICE=datadog-demo-run-go
 ENV DD_ENV=datadog-demo
 ENV DD_VERSION=1
 
+# this env var is needed for trace propagation to work properly in cloud run.
+# make sure to also set this variable for all Datadog instrumented downstream services.
```

Comment:
```suggestion
# ensure to set this variable for all Datadog-instrumented downstream services.
```

Reply:
Leaving just these suggestions here to avoid duplication

Reply:
fixed!